src/App.tsx

```diff
@@ -83,6 +85,23 @@ const App: FunctionComponent = function () {
     })
   }, [])
 
+  useEffect(() => {
+    if (env.LOG_PERF !== 'all' || Platform.OS === 'web') return
```

Comment:
on a un `App.web.tsx`, pas besoin de vérifier si on est en web 

```suggestion
    if (env.LOG_PERF !== 'all') return
```